scripts/release/shared-commands/print-prerelease-summary.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const clear = require('clear');
+const {join, relative} = require('path');
+const theme = require('../theme');
+
+module.exports = ({cwd}) => {
+  const publishPath = relative(
+    process.env.PWD,
+    join(__dirname, '../publish.js')
+  );
+
+  clear();
+
+  console.log(
+    theme`
+    {caution A release candidate has been prepared but you're not done yet!}
+
+    You can review the contents of this release in {path ./build/node_modules/}
+
+    {header Before publishing, please smoke test the packages!}
+
+    1. Open {path ./fixtures/packaging/babel-standalone/dev.html} in the browser.
```

Comment:
this doesn't work. the html assumes the files are in build/dist, but the script generates node_modules.

Reply:
Hm. At least some of the fixtures pull from `build/node_modules` (e.g. the Webpack fixture). I assumed they all did to be honest. Let me wipe out `build` and do a fresh pass at this to see what's up myself 😄

Reply:
I see the `build/dist` references. I'll update them to point into `build/node_modules` UMD folders!